paddle/pybind/pybind.cc

```diff
@@ -40,6 +42,23 @@ static size_t UniqueIntegerGenerator() {
   return generator.fetch_add(1);
 }
 
+std::once_flag gflags_init_flag;
```

Comment:
I remember that we have a convention from Google Style Guide -- global variables must be of POD type.

Reply:
@QiJune for the alternative, you can use pthread API for this global variable
 http://pubs.opengroup.org/onlinepubs/7908799/xsh/pthread_once.html

Reply:
@reyoung `pthread_once` is used as follows:
```
pthread_once(pthread_once_t *once_control, void (*init_routine)(void));
```
We have to pass a vector of string. So it's not suitable.

@wangkuiyi Yes, but [once_flag](http://en.cppreference.com/w/cpp/thread/once_flag) has no deconstructor. It behaves like a POD. And I have not found another better way to write this code yet.